api/router.py

```diff
@@ -10,7 +10,7 @@
 
 # App
 router = APIRouter(
-    prefix="/api",
+    prefix="/tasks",
```

Comment:
The frontend web app uses request forwarding so that /api/* goes to these urls. If this was /api then the frontend would reference it using /api/api/tasks which is not nice.